### PR TITLE
List peers as normalised objects - Closes #681

### LINKED ISSUE #681: List peers as normalised objects
There is the remaining issue introduced by merging the Web Socket protocol change branch regarding returning of non-normalized peers from public `api/peers` and internal `list` endpoints.

### Current behaviour
`__private.getByFilter` and `Peers.prototype.list` functions exposed for external usage returns peers as `Peer` instances always.

### Expected behaviour
Provide parameter for `__private.getByFilter` and `Peers.prototype.list` functions to customize normalization of returning peers.

### PR DESCRIPTION
**Pull request by MaciejBaj**
Distinguish usages of list and getByFilter functions in peers module between internal peers listing (normalised = false) and exposing the list for external usage (normalised = true). 
Provide normalisation parameter for peers listings.

Closes #681

### CODE REVIEW

**Comment by Isabello** [comment]
Running this on jenkins for you, didn't trigger automatically for some reason.

File: modules/transport.js [hunk 1]
```diff
@@ -374,7 +374,7 @@ Transport.prototype.onNewBlock = function (block, broadcast) {
 	if (broadcast) {
 		modules.system.update(function () {
 			if (!__private.broadcaster.maxRelays(block) && !modules.loader.syncing()) {
-				modules.peers.list({}, function (err, peers) {
+				modules.peers.list({normalized: false}, function (err, peers) {
```
Comment (4miners) [inline modules/transport.js]: Can we make `normalized: false` default, so no need to pass that param every time?

Comment (MaciejBaj) [inline modules/transport.js]: it is currently in place. Normalized set to `true` by default to remain previous functionalities:
https://github.com/LiskHQ/lisk/pull/683/files#diff-5ad295c030c66fe97f92104bd5d68cefR114

File: test/unit/modules/peers.js [hunk 2]
```diff
@@ -166,6 +167,52 @@ describe('peers', function () {
 		});
 	});
 
+	describe('list', function () {
+
+		beforeEach(function (done) {
+			removeAll(done);
+		});
+
+		it('should list empty peers list when no peers were insterted before', function (done) {
```
Comment (Isabello) [inline test/unit/modules/peers.js]: inserted

File: logic/broadcaster.js [hunk 3]
```diff
@@ -100,6 +100,7 @@ Broadcaster.prototype.bind = function (peers, transport, transactions) {
 Broadcaster.prototype.getPeers = function (params, cb) {
 	params.limit = params.limit || self.config.peerLimit;
 	params.broadhash = params.broadhash || null;
+	params.normalized = false;
```
Comment (4miners) [inline logic/broadcaster.js]: `normalize` instead of `normalized` is better

Comment (4miners) [inline logic/broadcaster.js]: ?

Comment (MaciejBaj) [inline logic/broadcaster.js]: normalized is ok for me as well :)

File: test/unit/modules/peers.js [hunk 4]
```diff
@@ -166,6 +167,52 @@ describe('peers', function () {
...
+
+		it('should list the inserted peer after insert', function (done) {
+			peers.update(randomPeer);
+			peers.list({}, function (err, __peers) {
+				expect(__peers).to.be.an('array').and.to.have.lengthOf(1);
+				done();
+			});
+		});
+
+		it('should list the inserted peer as Peer instance with normalized parameter set to false', function (done) {
```
Comment (diego-G) [inline test/unit/modules/peers.js]: `should list the inserted peer as object with normalized parameter set to false` ?

File: test/unit/modules/peers.js [hunk 5]
```diff
@@ -166,6 +167,52 @@ describe('peers', function () {
...
+
+		it('should list the inserted peer as object with normalized parameter set to true', function (done) {
+			peers.update(randomPeer);
+			peers.list({normalized: true}, function (err, __peers) {
+				expect(__peers[0]).to.be.an.instanceof(Object);
+				done();
+			});
+		});
+
+		it('should list the inserted peer as object without set normalized parameter', function (done) {
```
Comment (diego-G) [inline test/unit/modules/peers.js]: Repeated test?

File: test/unit/modules/peers.js (hunk 4 above)
Comment (MaciejBaj) [inline test/unit/modules/peers.js]: ?

File: test/unit/modules/peers.js (hunk 5 above)
Comment (MaciejBaj) [inline test/unit/modules/peers.js]: No, it tests the `list` function with and without `normalized` parameter

**Pull request merged**